.github/pull_request_template.md

```diff
@@ -2,6 +2,8 @@
 
 Describe in detail the changes you are proposing, and the rationale.
 
+Also, if it's practical, please provide an example config and commands for manually testing this change with Terraform CLI. (This isn't always possible, but it can enable better and faster reviews for your PR.)
```

Comment:
Perhaps we could add some headings so the contributor can just fill them in?

e.g. `#Old behaviour`, `#New behaviour`, `#How to test ##Example config ##Commands`

Reply:
@kmoe That's a good idea! 

I updated the PR to give that a try; it might have too many headers, but let's see what folks think.